cannon/docs/README.md

```diff
@@ -76,8 +72,8 @@ Nodes in this memory tree are combined as: `out = keccak256(left ++ right)`, whe
 and `left` and `right` are the 32-byte outputs of the respective sub-trees or the leaf values themselves.
 Individual leaf nodes are not hashed.
 
-The proof format is a concatenated byte array of 28 `bytes32`. The first `bytes32` is the leaf value,
-and the remaining 27 `bytes32` are the sibling nodes up till the root of the tree,
+The proof format is a concatenated byte array of 60 `bytes32`. The first `bytes32` is the leaf value,
+and the remaining 59 `bytes32` are the sibling nodes up till the root of the tree,
 along the branch of the leaf value, starting from the bottom.
 
 To verify the proof, start with the leaf value as `node`, and combine it with respective sibling values:
```

Comment:
There are a handful of references to the "4 bytes" [here](https://github.com/ethereum-optimism/optimism/blob/6197d796ea3603d2dd5c139caf5e53bac85135ed/cannon/docs/README.md?plain=1#L88-L107).  The instruction is still 4 bytes but the reads/write are 8-byte aligned and we read at most 8 bytes at a time.

Reply:
Oops, missed that. Let me know if what I wrote there now is correct.

Reply:
There's one more line [here](https://github.com/ethereum-optimism/optimism/blob/9134869d82e3132f6c826e054c884127041123a3/cannon/docs/README.md?plain=1#L107).